docs/_rpc/ns-debug.md

```diff
@@ -84,6 +84,25 @@ profile data to disk.
 | RPC     | `{"method": "debug_cpuProfile", "params": [string, number]}` |
 
 
+### debug_dbAncient
+
+Retrieves an ancient binary blob from the freezer. The freezer is a collection of append-only immutable files.
+They include types such as block headers, block bodies, receipts, etc.
```

Comment:
Since the `kind` is actually a keyword, I think it would be necessary to enumerate explicitly what it can be. Otherwise you hve no way of knowing what to pass there.